vuln/npm/457.json

```diff
@@ -0,0 +1,26 @@
+{
+  "id": 457,
+  "title": "Last Will message is not checked against authorization",
```

Comment:
title should be the weakness type, i.e: `Cross-site Scripting (XSS) - Generic`
ref: https://cwe.mitre.org/